src/questionGenerator.js

```diff
@@ -12,30 +12,62 @@ const questions = [
   'Death Row last meal - any meal, any restaurant in the world',
   'What is your favourite IDE?',
   'If you had to wear a shirt with one word on it for a year, what word would you choose?',
-  'If animals could talk, which one do you think would be the rudest?'
+  'If animals could talk, which one do you think would be the rudest?',
+  'Would you rather be able to speak any language fluently or speak with animals?',
+  'Where do you see yourself in five years from now?',
+  'What is your worst fear?',
+  'If your life was a movie, what would it be called?',
+  'Would you rather be an expert at only one thing or average at a lot of things?',
+  'Who was your role model growing up?',
+  'What is the happiest memory that you have?',
+  'What in life are you most passionate about?',
+  'Do you think it is possible to learn lessons without making mistakes first?',
+  'If you were invisible for a day, what would you do?',
+  'What makes you smile when you are having a bad day?',
+  'Would you rather live in the same place for the rest of your life, or have to move to a new country once a month for the next five years?',
+  'What is the most spontaneous thing you have ever done?',
+  'What was the best time of your life?',
+  'If you got kidnapped, what would you do that would be so annoying that it would make your kidnappers return you?',
+  'Would you rather live in a big city or the middle of nowhere for the rest of your life?',
+  'If you were to spend a day spoiling yourself, what would you do?',
+  'How much of your life would you guess you spend on your phone?',
+  'What keeps you up at night?',
+  'If you had to wear one outfit for the rest of your life, what would it be?',
+  'If you had a big decision to make, who would you talk to first?',
+  'What is one thing you wish you could change about yourself?',
+  'What is a life skill that you always wanted to learn and have never been able to?',
+  'Are there any lessons that you had to learn the hard way?',
+  'What is something that you cannot live without?',
+  'On a scale from one to tenhow would you rate this last week?',
```

Comment:
```suggestion
  'On a scale from one to ten how would you rate this last week?',
```

Reply:
Consider it fixed.

Reply:
Addressed code review change request(s) through:
-  [2258cef](https://github.com/mwall01/weekly-wall/pull/19/commits/2258cef70482c466aa79313ae2ff5a2fecd4ea01)